students/17th/team1/donggeunLEE/User/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.db.models.constraints import UniqueConstraint
+
+# Create your models here.
+
+class Userinfo(models.Model):
+    name         = models.CharField(max_length=20, unique=True)
+    Phone_number = models.IntegerField(unique=True)
```

Comment:
```suggestion
    phone_number = models.IntegerField(unique=True)
```
변수명은 소문자로, 그리고 전화번호를 인티저로 저장하면 0으로 시작하면 무시되므로, 적절하지 않습니다.

Reply:
``
phone_number = models.CharField(max_length=15, unique=Ture)
``
1. phone_number  - 소문자로 변경완료했습니다.
2. 정수형 필드인 IntegerField 대신 CharField로 변경했습니다.